From c0203b30124359554074375193196f4f682e9c4b Mon Sep 17 00:00:00 2001
From: AMG <andresmargar98@proton.me>
Date: Sat, 7 Feb 2026 12:51:17 +0000
Subject: [PATCH] 01-cmake

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 27864ad..3432be0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -203,6 +203,10 @@ set_property(TARGET CoreFoundation_x86_64 APPEND_STRING PROPERTY
   LINK_FLAGS " -Wl,-reexported_symbols_list,${CMAKE_CURRENT_SOURCE_DIR}/reexport_x86_64.exp \
 	-Wl,-alias,_OBJC_CLASS_\\$___NSCFConstantString,___CFConstantStringClassReference")
 endif (TARGET_x86_64)
+if (TARGET_ARM64)
+    set_property(TARGET CoreFoundation_arm64 APPEND_STRING PROPERTY
+       LINK_FLAGS " -Wl,-alias,_OBJC_CLASS_\\$___NSCFConstantString,___CFConstantStringClassReference")
+endif (TARGET_ARM64)
 
 set_source_files_properties(${cf_c_sources} PROPERTIES COMPILE_FLAGS "-x objective-c")
 
-- 
2.47.3

